test/e2e/flask/multichain-api/evm/wallet_sessionChanged.spec.ts

```diff
@@ -76,42 +76,43 @@ describe('Call `wallet_createSession`, then update the accounts and/or scopes in
          */
         await homePage.headerNavbar.openPermissionsPage();
         const permissionListPage = new PermissionListPage(driver);
-        await permissionListPage.checkPageIsLoaded();
-        await permissionListPage.openPermissionPageForSite(DAPP_HOST_ADDRESS);
-        const sitePermissionPage = new SitePermissionPage(driver);
-        await sitePermissionPage.checkPageIsLoaded(DAPP_HOST_ADDRESS);
-        await sitePermissionPage.editPermissionsForAccount(['Account 1']);
-        await sitePermissionPage.editPermissionsForNetwork(['Localhost 8545']);
+        await permissionListPage.checkPageIsLoaded(true);
+        // TODO: Update test for gator permissions page when it is implemented
```

Comment:
Not sure about disabling this test 🤔

Reply:
I think it would be preferable to show the current/previous page if you're planning to implement it in another PR, so we aren't in a broken state on the main branch at any point.